sakai-evaluation-api/src/java/org/sakaiproject/evaluation/beans/EvalBeanUtils.java

```diff
@@ -245,6 +245,18 @@ public void setEvaluationDefaults(EvalEvaluation eval, String evaluationType) {
         // make sure the dates are set
         Calendar calendar = new GregorianCalendar();
         Date now =  new Date();
+
+        // if default start hour is set, use it for start time (must be tomorrow as eval cannot start in the past).
+        Integer hour = (Integer) settings.get(EvalSettings.EVAL_DEFAULT_START_HOUR);
+        if (hour != null) {
+            // add 1 day to make it tomorrow
+            now.setTime(now.getTime() + 86400000L);
```

Comment:
Just an FYI, you could make this a little easier to read if you were to use Java 8's new and improved date/time handling:

```
Calendar cal = Calendar.getInstance();
cal.setTime( new Date() );
cal.add( Calendar.Day, 1 );
Date datePlusOneDay = cal.getTime();
```